spec/index.bs

```diff
@@ -4403,7 +4391,7 @@ Issue(gpuweb/gpuweb#354): Finish defining multithreading API and add `[Serializa
 
 <script type=idl>
 dictionary GPUShaderModuleCompilationHint {
-    required GPUPipelineLayout layout;
+    required GPUPipelineLayoutRequirements layout;
```

Comment:
Does this usage allow `"auto"`? If not, maybe we should consider defining an second `PipelineLayout` variant that doesn't include the union.

Reply:
Yes, it does. The layout can be computed from only the entry point; none of the rest of the pipeline descriptor is needed. So I think it's equally valid to use "auto" in a compilation hint as in pipeline creation.